scheduler/roles/cluster_validation/tasks/set_facts.yml

```diff
@@ -63,7 +63,7 @@
       ansible.builtin.set_fact:
         selected_cluster: >-
           {{ vars[cluster_var_name]
-              | selectattr('deployment', 'equalto', 'true')
+              | selectattr('deployment', 'equalto', true)
```

Comment:
does this work with any kind of combination of value true, True or string 'true'?